assets/production-bestpractices.md

```diff
@@ -0,0 +1,89 @@
+
+# StorageOS production installation
+
+*Installation with DaemonSets on Kubernetes*
+
+StorageOS tries to keep the installation process simple for users, requiring only minimal configuration. However, more options are available if you are deploying in complex environments, or where tuning for availability and performance is desired.  This guide discusses deployment options to consider when running in production.
+
+Checkout the StorageOS Helm [chart](https://github.com/storageos/helm-chart) for an easy automated installation.
```

Comment:
Checkout => Check out
Link the full "StorageOS Helm Chart", not just "chart".

Reply:
done